sirepo/template/raydata.py

```diff
@@ -4,9 +4,13 @@
 :copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
+from pykern import pkcompat
```

Comment:
not used

Reply:
34d5c6d1b0aa5be7fb841e47a6fbf0fc34ecd9e9